README.md

```diff
@@ -1 +1,69 @@
-# EzSystemsPrivacyCookieBundle
+# Privacy Cookie Bundle
+
+This bundle adds privacy cookie banner into eZ Publish / eZ Platform.
+
+## Requirements
+
+- eZPublish 5.4/2014.11 or above
```

Comment:
With former comment of removing reference to content id ,this can potentially be removed to rather say any Symfony 2.x application, including eZ Platform and eZ Publish 5.x